SPECS/xcp-ng-generic-lib.spec

```diff
@@ -29,7 +32,7 @@ cd build
 %files
 %license LICENSE
 %{_libdir}/libxcp-ng-generic.so.1
-%{_libdir}/libxcp-ng-generic.so.1.0.0
+%{_libdir}/libxcp-ng-generic.so.1.1.0
```

Comment:
If the suffix always matches the version, you may replace it with %{version} to avoid having to manually update it.